utils/Constants.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+var Constants map[string]string = map[string]string{
```

Comment:
nitpick: a variable named Constants is a little confusing.

Reply:
Not able to think of anything else, can you suggest some name please. @dev-gv

Reply:
Changed it as https://github.com/Real-Dev-Squad/feature-flag-backend/pull/27#discussion_r1187645347